redhat-ipaas-dev-single-tenant.yml

```diff
@@ -95,6 +95,11 @@ parameters:
   name: POSTGRESQL_VOLUME_CAPACITY
   required: true
   value: 1Gi
+  - description: Enables test-support endpoint on backend API
+    displayName: Test Support Enabled
+    name: TEST_SUPPORT_ENABLED
+    required: true
+    value: "true"
```

Comment:
@dsimansk would it be difficult or too tedious for you when this would be disabled by default even for the dev templates ? (so that you would need to enable it explicitly for testing). I'm asking because that would even reduce the risk when the wrong templates are leaked or distributed (you never know)

But if it put a too much of burden on you, we can also leave this.